app/controllers/game.js

```diff
@@ -230,6 +239,25 @@ export default Ember.Controller.extend({
 
     toggleChat: function () {
       this.toggleProperty('showChat');
+    },
+
+    /**
+      * Increment or decrement life
+      */
+    changeLife: function (delta) {
+      var participant = this.get('participant');
+      var life = participant.get('life');
+      participant.set('life', life += delta);
+      this.get('model').save();
+    },
+
+    /**
+      * A player has called it quits!
+      */
+    endGame: function () {
+      var game = this.get('model');
+      game.set('status', 'ended');
+      game.save().then(() => this.transitionToRoute('games.list'));
```

Comment:
Due to the `statusChanged` observer, looks like the `transitionToRoute` might not be needed here.